config/config-example.toml

```diff
@@ -280,41 +426,73 @@ gossip_request_timeout = '30sec'
 # from a peer which gossiped information about that data to this node.
 get_remainder_timeout = '5sec'
 
+# The timeout duration for a newly-received, gossiped item to be validated and stored by another
+# component before the gossiper abandons waiting to gossip the item onwards.
+validate_and_store_timeout = '1min'
 
-# =================================
-# Configuration options for fetcher
-# =================================
+
+# ===============================================
+# Configuration options for the block accumulator
+# ===============================================
+[block_accumulator]
+
+# Block height difference threshold for starting to execute the blocks.
+attempt_execution_threshold = 3
+
+# Accepted time interval for inactivity in block accumulator.
+dead_air_interval = '3min'
+
+# Time after which the block acceptors are considered old and can be purged.
+purge_interval = '1min'
```

Comment:
This is `'5min'` in resources/local/config.toml - I'd suggest changing this one to 5 mins if that's what we've been testing with.